x/oracle/types/params.go

```diff
@@ -23,7 +23,7 @@ var (
 // Default parameter values
 const (
 	DefaultVotePeriod               = BlocksPerMinute / 2 // 30 seconds
-	DefaultSlashWindow              = BlocksPerWeek       // window for a week
+	DefaultSlashWindow              = BlocksPerDay        // window for a day
```

Comment:
Maybe instead of changing it here, we should use non-default genesis? The idea would be to modify genesis either:
* before running `collect-gentx` using a script (so that would require one more step when setting up a testnet)
* or adding a switch / genesis modification in migration code (`app/upgrades.go`)
WDYT?

Reply:
Editing someone else comment to add response looks odd :P

Reply:
Wtf? I did this on the app, I think that's why 😓